controllers/grid/catalogEntry/form/PublicationFormatForm.inc.php

```diff
@@ -188,6 +205,36 @@ function execute(...$functionParams) {
 			SubmissionLog::logEvent(Application::get()->getRequest(), $this->getMonograph(), SUBMISSION_LOG_PUBLICATION_FORMAT_CREATE, 'submission.event.publicationFormatCreated', array('formatName' => $publicationFormat->getLocalizedName()));
 		}
 
+
+		// Save ISBN identification code
+
+		$identificationCodeDao = DAORegistry::getDAO('IdentificationCodeDAO'); /* @var $identificationCodeDao IdentificationCodeDAO */
+
+		// Remove existing ISBN-10 or ISBN-13 code 
+		$identificationCodes = $identificationCodeDao->getByPublicationFormatId($representationId);
+		while ($identificationCode = $identificationCodes->next()) {
+			if ($identificationCode->getCode() == "02" || $identificationCode->getCode() == "15") {
+				$identificationCodeDao->deleteById($identificationCode->getId());					
+			}
+		}
+
+		// Add new ISBN codes
+		if ($this->getData('isbn10') || $this->getData('isbn13')){
+
+			if ($this->getData('isbn10')) { $isbnValues['02'] = $this->getData('isbn10'); }
```

Comment:
The array `$isbnValues` needs to be instantiated as an array first, or it will produce a PHP notice.

Reply:
ok